composer.json

```diff
@@ -19,7 +19,7 @@
         "cs:check": "php-cs-fixer fix --dry-run --diff",
         "cs:fix": "php-cs-fixer fix",
         "psalm": "psalm.phar --threads=1",
-		"psalm:update-baseline": "psalm.phar --threads=1 --update-baseline --set-baseline=tests/psalm-baseline.xml",
+		"psalm:update-baseline": "psalm.phar --threads=1 --update-baseline",
```

Comment:
actually the removed part is really updating the baseline

Reply:
no, the --set-baseline is dangerous and will not warm you when you have regression

--update-baseline alone has the correct behavior and will just remove the stuff that as been fixed